src/styles/blocks/main.scss

```diff
@@ -0,0 +1,146 @@
+.main {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 944px;
+  margin: 0 auto 70px;
+}
+
+.product-card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 404px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  margin-top: 50px;
+}
+
+.main > :not(:nth-child(4n)) {
+  margin-right: 48px;
+}
```

Comment:
Don't style margins between cards like this.
This way is very unreliable. `main` block may not know what's inside of it.